bin/_help

```diff
@@ -16,6 +16,9 @@ function help() {
     echo "console         |  Run user supplied programs inside a crucible wrapper for logging purposes"
     echo "wrapper         |  Run a crucible subproject command directly, within a crucible container"
     echo "get             |  Get either a result summary (get result) or any metric (get metric)"
+    echo "index           |  Index a result which is already post-processed with run-result-dir argument immediately following 'index'"
+    echo "postprocess     |  Post-process (and index) a result with run-result-dir argument immediately following 'index'"
+    echo "reindex         |  Completely delete and init ES, then post-process and index all runs found in $var_crucible/run"
```

Comment:
Should this change slightly to `index [all]` and `postprocess [all]`?

Given the fact that we currently have `index` and `postprocess` where `postprocess` does both, it seems odd that `reindex` would also do both while `index` does not.  If it was switched to both arguments having an optional `all` parameter then the behavior between operating on one or many results could be the same.

I'm just thinking that consistency will make it easier to use...

Reply:
Yeah, I figured we would refine this a bit, but I wanted something in there for now in case others need to reindex everything.